perseo-main/src/main/java/com/telefonica/iot/perseo/RulesServlet.java

```diff
@@ -108,16 +112,21 @@ protected void doPost(HttpServletRequest request, HttpServletResponse response)
         Utils.putCorrelatorAndTrans(request);
         response.setCharacterEncoding("UTF-8");
         response.setContentType("application/json;charset=UTF-8");
-        PrintWriter out = response.getWriter();
-        String body = Utils.getBodyAsString(request);
 
-        // Save temporary rules, not triggered by events external to the core
-        TimeRulesStore.getInstance().saveTimeRules(body);
+        try {
+			String body = Utils.getBodyAsString(request);
```

Comment:
Wrong indentation

Reply:
Fixed in 61b8a4c9c7fe92f547917fd56b451a77f39db5f5